geospatial/dted.ksy

```diff
@@ -0,0 +1,399 @@
+        type: ascii_int_ssss
+      - id:  partial_cell_indicator
+        type: str
+        size: 2
+        encoding: ASCII
+      - id:  reserved_for_nima_use_only
+        type: str
+        size: 101
+        encoding: ASCII
+      - id:  reserved_for_producing_nation_use_only
+        type: str
+        size: 100
+        encoding: ASCII
+      - id:  reserved_for_free_text_comments
+        type: str
+        size: 155
+        encoding: ASCII
+  acc_header:
+    seq:
+      - id: recognition_sentinel
+        contents: ACC
+      - id: absolute_horizontal_accuracy
+        type: ascii_int_ssss
+      - id: absolute_vertical_accuracy
+        type: ascii_int_ssss
+      - id: relative_horizontal_accuracy_of_product_in_meters
+        type: ascii_int_ssss
+      - id: relative_vertical_accuracy_or_product_in_meters
+        type: ascii_int_ssss
+      - id: reserved_for_future_use1
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: reserved_for_nima_use_only1
+        type: str
+        size: 1
+        encoding: ASCII
+      - id: reserved_for_future_use2
+        type: str
+        size: 31
+        encoding: ASCII
+      - id: multiple_accuracy_outline_flag
+        type: str
+        size: 2
+        encoding: ASCII
+      - id: subregion
+        type: acc_subregion
+        repeat: expr
+        repeat-expr: 9
+      - id: reserved_for_nima_use_only2
+        type: str
+        size: 18
+        encoding: ASCII
+      - id: reserved_for_future_use3
+        type: str
+        size: 69
+        encoding: ASCII
+  acc_subregion:
+    seq:
+      - id: absolute_horizontal_accuracy_or_subregion_in_meters
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: absolute_vertical_accuracy_or_subregion_in_meters
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: relative_horizontal_accuracy_of_subregion_in_meters
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: relative_vertical_accuracy_of_subregion_in_meters
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: number_of_coordinates_in_accuracy_subregion_outline
+        type: str
+        size: 2
+        encoding: ASCII
+      - id: coordinate_pair
+        type: coordinate_pair_ddmmss_sh
+        repeat: expr
+        repeat-expr: 14
+  data_record:
+    seq:
+      - id: recognition_sentinal
+        type: u1
+      - id: data_block_count
+        type: b24
+      - id: logitude_count
+        type: u2
+      - id: latitude_count
+        type: u2
+      - id: elevations
+        type: data_record_elevation
+        repeat: expr
+        repeat-expr: _parent.uhl.number_of_latitude_points.value
+      - id: checksum
+        type: s4
+  data_record_elevation:
+    seq:
+      - id: is_negative
+        type: b1
+      - id: value
+        type: b15
+    instances:
+      meters:
+        value: value * (is_negative? -1 : 1)
+  ascii_int_ssss:
```

Comment:
either just `ascii_int` or `ascii_int(4)`. And maybe not `ascii` but `text`?

Reply:
the spec explicitly mentions that the text is ascii so I'm inclined more towards ascii_int. I'll update accordingly.